cloudslang-content-verifier/src/main/java/io/cloudslang/lang/tools/build/validation/StaticValidatorImpl.java

```diff
@@ -0,0 +1,91 @@
+/*
+ * (c) Copyright 2016 Hewlett-Packard Enterprise Development Company, L.P.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Apache License v2.0 which accompany this distribution.
+ *
+ * The Apache License is available at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ */
+package io.cloudslang.lang.tools.build.validation;
+
+import io.cloudslang.lang.compiler.Extension;
+import io.cloudslang.lang.compiler.modeller.model.Executable;
+import io.cloudslang.lang.compiler.modeller.model.Metadata;
+import io.cloudslang.lang.entities.bindings.InOutParam;
+import org.apache.commons.collections4.ListUtils;
+import org.apache.commons.lang.Validate;
+import org.springframework.stereotype.Component;
+
+import java.io.File;
+import java.util.List;
+import java.util.Map;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+/**
+ * Created by bancl on 8/30/2016.
+ */
+@Component
+public class StaticValidatorImpl implements StaticValidator {
+
+    @Override
+    public void validateSlangFile(File slangFile, Executable executable, Metadata metadata, Boolean shouldValidateDescription){
```

Comment:
prefer primitive types, Boolean for `shouldValidateDescription `implies that we don't know at calling time the fact that we validate or not descriptions